Library/Homebrew/os/mac/xcode.rb

```diff
@@ -171,7 +173,8 @@ def detect_version_from_clang_version
         when 81      then "8.3"
         when 90      then "9.2"
         when 91      then "9.3"
-        else              "9.3"
+        when 10001   then "10.0"
```

Comment:
With the CLT, and the patch I mentioned earlier for `clang_version`, I'm seeing a value of `100` here, not `10001`. What string are you getting for `clang --version`? Are you using Xcode, and what version?

Reply:
> I'm seeing a value of 100 here

That would be more like what I expected, to be honest! I assumed something was going on when `10001` popped out but I had to go grab some sleep so pushed it for review as-is in case someone else had a more sensible answer.

> What string are you getting for clang --version?

```
Apple LLVM version 10.0.0 (clang-1000.10.25.5)
Target: x86_64-apple-darwin17.7.0
Thread model: posix
InstalledDir: /Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

> Are you using Xcode, and what version?

Aye, 10.0.